activity_calendar/admin.py

```diff
@@ -16,10 +16,28 @@ def is_recurring(self, obj):
 admin.site.register(Activity, ActivityAdmin)
 
 
+
+
 @admin.register(ActivityMoment)
 class ActivityMomentAdmin(admin.ModelAdmin):
+    list_filter = ['parent_activity']
     fields = ['parent_activity', 'recurrence_id', 'local_description', 'local_location', 'local_max_participants']
 
+    def activity_moment_has_changes(obj):
+        """ Check if this ActivityModel has any data it overwrites """
+        for field in obj._meta.local_fields:
+            # Check for all local_ fields if it is None, if not, it overwrites an attribute
+            if field.name.startswith("local_"):
+                local_value = getattr(obj, field.name)
+                if local_value is not None and local_value != "":
+                    return True
+        return False
+    activity_moment_has_changes.short_description = 'Is tweaked'
```

Comment:
We can use .boolean to have it display a checkmark/cross instead
```suggestion
    activity_moment_has_changes.boolean = True
    activity_moment_has_changes.short_description = 'Is tweaked'
```